### PR TITLE
[10] UI Job Information on Job Details

### LINKED ISSUE #10: [Android] [UI] As a user, I can see the job information in the job details page
## Why

As we're creating a demo to show job detail, the user should be able to read the details of a job, including price, location, time, etc before deciding to accept the jobs or not.

## Acceptance Criteria

On the job detail screen, we need to implement a scrollable area which includes:
- the price section
  - Show `Price by area` in gray
  - Show price info `80-95 Baht` in green
- the date-time section (no expandable)
  - Show section title in black
  - Show the date in black
  - Show the time in gray
- the plot info section (no expandable)
  - Show the section title in black
  - Show the distance and area info in gray for title and green for value
  - Show the address and crop type info in gray for the title and black for the value
- the message section (no expandable)
  - Show the title and value in black and gray

## Design

image

## Resources

- black: `#171923`
- gray: `#A0AEC0`
- green: `#2BC65D`

### PR DESCRIPTION
**Pull request by lydiasama**
#10

## What happened 👀

Add Job Information on Job Details screen
 
## Insight 📝

On the job detail screen, we need to implement a scrollable area which includes:
- the price section
  - Show `Price by area` in gray
  - Show price info `80-95 Baht` in green
- the date-time section (no expandable)
  - Show section title in black
  - Show the date in black
  - Show the time in gray
- the plot info section (no expandable)
  - Show the section title in black
  - Show the distance and area info in gray for title and green for value
  - Show the address and crop type info in gray for the title and black for the value
- the message section (no expandable)
  - Show the title and value in black and gray

### CODE REVIEW

File: KMM_Showcase/src/main/res/layout/fragment_job_details.xml [hunk 1]
```diff
@@ -2,7 +2,49 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:paddingTop="@dimen/margin_default">
+
+    <TextView
+        android:id="@+id/tvJotDetailPriceByAreaTitle"
```
Comment (luongvo) [inline KMM_Showcase/src/main/res/layout/fragment_job_details.xml]: Oops I think it should be `Job` 😂 

```suggestion
        android:id="@+id/tvJobDetailPriceByAreaTitle"
```

same for all ids below

Comment (lydiasama) [reply KMM_Showcase/src/main/res/layout/fragment_job_details.xml]: Oh wrong copy-paste, let me update. 😂

**Comment by luongvo** [comment]
@lydiasama I don't see Plot Info and Message sections from the code 😕

**Comment by lydiasama** [comment]
@luongvo  It seems the commits are not pushed yet. 🤔  Today I have a problem with my AS. Actually, the naming issue was resolved in the second commit too.

**Comment by lydiasama** [comment]
@luongvo Okay, the second commit is here now. 🎉

File: KMM_Showcase/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
     <string name="job_details_price_label">Price</string>
     <string name="job_details_price_value">80-95 Baht</string>
-    <string name="action_accept">Accept</string>
+    <string name="job_details_price_by_area">Price by area</string>
+    <string name="job_details_date_time">Date time</string>
+    <string name="job_details_plot_information">Plot Information</string>
+    <string name="job_details_plot_info_distance_label">Distance</string>
+    <string name="job_details_plot_info_area_label">Area</string>
+    <string name="job_details_plot_info_distance_value">33 km</string>
+    <string name="job_details_plot_info_area_value">70 m2</string>
```
Comment (luongvo) [inline KMM_Showcase/src/main/res/values/strings.xml]: they are values from API response, we could hardcode in the layout and replace them later

File: KMM_Showcase/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="action_accept">Accept</string>
     <string name="job_details_title">Job Detail</string>
     <string name="job_details_bottom_panel_title">Get this job to know more about farmer contact information</string>
     <string name="job_details_price_label">Price</string>
     <string name="job_details_price_value">80-95 Baht</string>
```
Comment (luongvo) [inline KMM_Showcase/src/main/res/values/strings.xml]: value from API response, we could hardcode in the layout and replace it later

**Review by luongvo — COMMENTED** [review]
rest lgtm

**Review by luongvo — APPROVED** [review]
LGTM, @hoangnguyen92dn please review

**Comment by lydiasama** [comment]
@luongvo Oh, At first, I intended to update it to `%s m2`

**Comment by luongvo** [comment]
> @luongvo Oh, At first, I intended to update it to `%s m2`

Yes, we can consider it later then 😄

**Review by hoangnguyen92dn — APPROVED** [review]

**Pull request merged**